components/repositories/AccountView.tsx

```diff
@@ -534,7 +534,7 @@ function Details({
         )}
         {hcapDetail && (
           <DataField value={ip} label="Hcaptcha">
-            {Object.entries(hcapDetail).map(([property, value]) => (
+            {hcapDetail.map(({ property, value }) => (
```

Comment:
.map will fail if `hcapDetail` undefined so you can either fallback to `[]` up top or just `?.` here.

Reply:
ah good looks 👍